exercises/house/src/test/java/HouseTest.java

```diff
@@ -0,0 +1,344 @@
+                "This is the man all tattered and torn\n" +
+                "that kissed the maiden all forlorn\n" +
+                "that milked the cow with the crumpled horn\n" +
+                "that tossed the dog\n" +
+                "that worried the cat\n" +
+                "that killed the rat\n" +
+                "that ate the malt\n" +
+                "that lay in the house that Jack built.\n" +
+                "\n" +
+                "This is the priest all shaven and shorn\n" +
+                "that married the man all tattered and torn\n" +
+                "that kissed the maiden all forlorn\n" +
+                "that milked the cow with the crumpled horn\n" +
+                "that tossed the dog\n" +
+                "that worried the cat\n" +
+                "that killed the rat\n" +
+                "that ate the malt\n" +
+                "that lay in the house that Jack built.\n" +
+                "\n" +
+                "This is the rooster that crowed in the morn\n" +
+                "that woke the priest all shaven and shorn\n" +
+                "that married the man all tattered and torn\n" +
+                "that kissed the maiden all forlorn\n" +
+                "that milked the cow with the crumpled horn\n" +
+                "that tossed the dog\n" +
+                "that worried the cat\n" +
+                "that killed the rat\n" +
+                "that ate the malt\n" +
+                "that lay in the house that Jack built.\n" +
+                "\n" +
+                "This is the farmer sowing his corn\n" +
+                "that kept the rooster that crowed in the morn\n" +
+                "that woke the priest all shaven and shorn\n" +
+                "that married the man all tattered and torn\n" +
+                "that kissed the maiden all forlorn\n" +
+                "that milked the cow with the crumpled horn\n" +
+                "that tossed the dog\n" +
+                "that worried the cat\n" +
+                "that killed the rat\n" +
+                "that ate the malt\n" +
+                "that lay in the house that Jack built.\n" +
+                "\n" +
+                "This is the horse and the hound and the horn\n" +
+                "that belonged to the farmer sowing his corn\n" +
+                "that kept the rooster that crowed in the morn\n" +
+                "that woke the priest all shaven and shorn\n" +
+                "that married the man all tattered and torn\n" +
+                "that kissed the maiden all forlorn\n" +
+                "that milked the cow with the crumpled horn\n" +
+                "that tossed the dog\n" +
+                "that worried the cat\n" +
+                "that killed the rat\n" +
+                "that ate the malt\n" +
+                "that lay in the house that Jack built.";
+
+        int startVerse = 1;
+        int endVerse = 12;
+
+        assertEquals(expected, house.verses(startVerse, endVerse));
```

Comment:
Maybe having a method called `sing` or `singWholeSong` or something might be useful? So that the caller doesn't have to know that there are 12 verses?